src/__tests__/benchmark.test.js

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import App from '../App';
+import { HashRouter as Router } from 'react-router-dom';
+import {
+  render,
+  waitFor,
+  queryByText,
+  findByText,
+  screen,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+// Since I updated dependencies in package.json, this seems necessary.
+beforeEach(() => {
+  Object.defineProperty(window, 'matchMedia', {
+    writable: true,
+    value: jest.fn().mockImplementation(query => ({
+      matches: false,
+      media: query,
+      onchange: null,
+      addListener: jest.fn(), // deprecated
+      removeListener: jest.fn(), // deprecated
+      addEventListener: jest.fn(),
+      removeEventListener: jest.fn(),
+      dispatchEvent: jest.fn(),
+    })),
+  });
+});
+
+/**
+ * Return true if given DOM plot element contains all given texts.
+ * @param plot - DOM element containing a plot
+ * @param texts - texts (strings) to search
+ * @return {boolean} - true if plot contains all texts
+ */
+function plotHasTexts(plot, texts) {
+  for (let text of texts) {
+    if (queryByText(plot, text) === null) return false;
+  }
+  return true;
+}
+
+/**
+ * Check that we find only 1 plot containing all given texts.
+ * @param texts - texts to search
+ */
+async function lookupPlot(...texts) {
+  await waitFor(
+    () => {
+      const plots = document.querySelectorAll('.orion-plot');
+      const filtered = [];
+      for (let plot of plots.values()) {
+        if (plotHasTexts(plot, texts)) {
+          filtered.push(plot);
+        }
+      }
+      expect(filtered.length).toBe(1);
+    },
+    { interval: 1000, timeout: 20000 }
+  );
+}
+
+test('Test select benchmark', async () => {
```

Comment:
It would be good to add tests for selections of assessments, tasks and algorithms as well.